keepassxc-browser/content/autocomplete.js

```diff
@@ -0,0 +1,156 @@
+'use strict'
+
+var kpxcAutocomplete = {};
+kpxcAutocomplete.elements = [];
+
+kpxcAutocomplete.create = function(input, showListInstantly = false) {
+    let _index = -1;
+
+    input.addEventListener('click', function(e) {
```

Comment:
I have, but decided to keep the current behaviour, for now. When selecting the input using `Tab` key, the autocomplete list can still be opened using `Delete` or `Backspace`. I added these shortcuts to make the user experience more friendly for people who like to use the keyboard instead of mouse.